content/en/blogs/kubesphere-certificates.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Come on! Your Exclusive Certificate is Ready'
+tag: 'KubeSphere,Community'
+keyword: 'Community, Open Source, Contribution, KubeSphere'
+description: 'KubeSphere Community Steering Committee would like to express gratitude to KubeSphere Member, KubeSphere Ambassador, KubeSphere Talented Speaker, and KubeSphere Contributor in 2020'
+createTime: '2021-02-11'
+author: 'Feynman Zhou'
+snapshot: '/images/blogs/en/meetup-2020/kubesphere-meetup.png'
+---
+
+
+
+It’s highly appreciated that the KubeSphere community has nearly 100 contributors  by February 2021. Voted and approved by the KubeSphere Community Technical Committee and the KubeSphere Steering Committee, 10 Ambassadors, 15 Talented Speakers, 22 outstanding contributors, and 6 active contributors with remarkable contributions were selected throughout the year.  
+
+
+
+## Vote of Thanks
+
+KubeSphere Community Steering Committee would like to express gratitude to KubeSphere Member, KubeSphere Ambassador, KubeSphere Talented Speaker, and KubeSphere Contributor, and extend the sincerest greetings to all the members who have participated in the open-source contribution of the KubeSphere community. Therefore, we prepare an exclusive gift for you.
```

Comment:
KubeSphere Member, KubeSphere Ambassador, KubeSphere Talented Speaker, and KubeSphere Contributor
->
KubeSphere Members, KubeSphere Ambassadors, KubeSphere Talented Speakers, and KubeSphere Contributors

Reply:
who have participated in the open-source contribution of the KubeSphere community
->
who participated in the open-source contribution to the KubeSphere community